src/Spatial/Extensions/DoubleExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MathNet.Spatial.Extensions
+{
+    internal static class DoubleExtensions
+    {
+        internal static bool IsNearlyEqualTo(this double d, double reference, double tolerance)
```

Comment:
This is not being used anywhere. If so, it should be removed.

Reply:
Thanks for the comment.
I agree with your opinion, that is, I will remove this.

Reply:
done in the commit https://github.com/f-frhs/mathnet-spatial/commit/2d222ac60492d5ca6a2028cb6a2c7a25193ce640